lib/graphql/subscriptions/anycable_subscriptions.rb

```diff
@@ -243,6 +265,33 @@ def fetch_channel_istate(channel)
       def redis_key(prefix)
         "#{config.redis_prefix}-#{prefix}"
       end
+
+      def executor_class_job
+        custom_class = config.async_broadcasting["class"]
+
+        return Adapters::BaseJob unless custom_class
+
+        Object.const_get(config.async_broadcasting["class"])
```

Comment:
The job class is inferred in Rails context where we can use `#safe_constantize`

Reply:
Thanks!
But for  `job_class`, I put `#constantize` instead of` #safe_constantize`, because I think that `#constantize` will be useful for cases when the user puts anything wrong to param `job_class`.
And `constantize` will raise an error and we will see it immediately

But, if I am wrong, I will fix it